src/pages/Overview/OverviewPage.tsx

```diff
@@ -478,6 +491,24 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
       // Actions can be personalized by namespace
       // i.e. Add or Remove depending of presence of a flag/label
       const actions = this.showActions();
+      if (serverConfig.istioInjectionAction) {
```

Comment:
Perhaps is personal style, but what about moving this `istioInjectionAction` into a specific method like `appendInjectionAction(actions, ns)`. The render method is really compact so far, it doesn't know about the implementation of the action list, so it looks a bit cleaner to have this logic in a different method.